docs/developers/index.html

```diff
@@ -46,6 +47,9 @@ <h3>WoT Implementations</h3>
 		</li>
 		<li>
 			<a href="https://github.com/node-red/node-red-nodegen">Node generator</a> (Generate a WoT Consumer Node for <a href="https://nodered.org/">Node-RED</a> from TD)
+			<ul>
+				<li>See <a href="wot_nodegen_20200507.pptx">short introduction material for Node Generator</a></li>
```

Comment:
I made a mp4 version of slides in PR #61.
Could you change this link to 'wot_nodegen_20200507.mp4'?

Reply:
I replaced the powerpoint slides with the video.

Note: We need to check whether mp4 can be played directly in every browser OR we need to wrap it in `<video>` tag or so (I checked Firefox and it worked).